src/EnergyPlus/DXCoils.cc

```diff
@@ -7500,8 +7492,8 @@ namespace DXCoils {
 				OutdoorWetBulb = OutWetBulbTemp;
 			}
 			if ( DXCoil( DXCoilNum ).IsSecondaryDXCoilInZone ) {
-				OutdoorDryBulb = Node( DXCoil( DXCoilNum ).SecZoneAirNodeNum ).Temp;
-				OutdoorHumRat = Node( DXCoil( DXCoilNum ).SecZoneAirNodeNum ).HumRat;
+				OutdoorDryBulb = ZT( DXCoil( DXCoilNum ).SecZonePtr );
+				OutdoorHumRat = ZoneAirHumRat( DXCoil( DXCoilNum ).SecZonePtr );
 				OutdoorWetBulb = DXCoil( DXCoilNum ).EvapInletWetBulb;
 				OutdoorPressure = OutBaroPress;
 			}
```

Comment:
I could not get rid of the if (IsSecondaryDXCoilInZone) since an uncontrolled zone has a 0 for zone air node in the ZoneEquipConfig array. I did not want to try to "set" that node at this point. Maybe it would have been easy to do so, I didn't try. Maybe a new issue to clean this up entirely? Or should I just bite the bullet now?